### DISCUSSION TITLE
빌드잇 맵 불러올 때 500 Internal Server Error 발생

### DISCUSSION THREAD

**Discussion by kaav846** [naverz/zepeto-studio-kor#131] [Issue Report (Legacy)]
어제까지 문제없이 빌드잇 맵을 제페토 게임스튜디오로 불러올 수 있었는데,
지금은 ZEPETO ->  bulidit -> Map Loader 를 눌러도 아무 창이 뜨지 않고 하위 에러가 발생합니다.

Exception: HTTP/1.1 500 Internal Server Error
ZEPETO.Buildit.Loader.Authentication+<Login>d__15.MoveNext () (at <66ed890cd1c7499bb4c89cccdf316f98>:0)
Unity.EditorCoroutines.Editor.EditorCoroutine.ProcessIEnumeratorRecursive (System.Collections.IEnumerator enumerator) (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:140)
Unity.EditorCoroutines.Editor.EditorCoroutine.MoveNext () (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:115)
UnityEditor.EditorApplication.Internal_CallUpdateFunctions () (at <83a38f5397d94e36a22efa3369c82390>:0)

=====================================================================================

HTTP/1.1 500 Internal Server Error
UnityEngine.Debug:LogError (object)
ZEPETO.Buildit.Loader.BuilditWindow/<WaitMapListRequest>d__12:MoveNext ()
Unity.EditorCoroutines.Editor.EditorCoroutine:ProcessIEnumeratorRecursive (System.Collections.IEnumerator) (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:148)
Unity.EditorCoroutines.Editor.EditorCoroutine:MoveNext () (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:115)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()

확인 부탁드리겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전 1.1.5 에서도 지속적으로 나타나는지 확인 부탁드립니다.
